www/src/components/container.js

```diff
@@ -16,6 +16,7 @@ export default ({ children, className, hasSideBar = true }) => (
       [presets.Tablet]: {
         paddingBottom: rhythm(1.5),
       },
+      ...css,
```

Comment:
This isn't necessary as the Glamor plugin we add converts the `css` prop into a `className` before passing it in which is already being added to the div.